armada_command/command_ssh.py

```diff
@@ -50,10 +49,15 @@ def command_ssh(args):
 
     docker_key_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'keys/docker.key')
 
+    # TODO: Should invert this condition and replace this with -t command flag?
+    tty = '-t'
     if args.command:
-        command = 'sudo ' + ' '.join(args.command)
+        command = ' '.join(args.command)
+        if command.startswith('bash'):
+            tty = ''
     else:
-        command = 'sudo -i'
-    ssh_command = ('ssh -t -p {ssh_port} -i {docker_key_file} -o StrictHostKeyChecking=no docker@{ssh_host} '
-                   '"{command}"').format(**locals())
+        command = 'bash'
+
+    exec_command = 'docker exec -i {tty} {container_id} {command}'.format(**locals())
```

Comment:
Unfortunately `docker exec` does not propagate env var TERM, as `ssh` does, so when I try to run (inside the container) some graphical program, like `mc` I get:

    The TERM environment variable is unset!

We should set TERM to the value of $TERM on Armada ship.